godtools/Views/TractElements/TractCards+ChildSetup.swift

```diff
@@ -47,8 +51,12 @@ extension TractCards {
             let element = TractCard(data: dictionary, startOnY: yPosition, parent: self, elementNumber: elementNumber)
             element.yDownPosition = yDownPosition - TractPage.navbarHeight
             element.cardProperties().cardNumber = cardNumber
+            let alphaCode = element.cardProperties().cardNumber.convertToLetter()
+            element.cardProperties().cardIdName = "\(alphaCode)"
```

Comment:
Does this need to be tracked as a separate property? I'm not seeing how it's being used.

Reply:
It looks like we could just call `cardNumber.convertToLetter()` in TractCards+Actions , then cardIdName isn't necessary.

Reply:
I'm now using cardLetterName and getting from cardNumber.convertToLetter()